src/backend/settings.py

```diff
@@ -522,3 +522,12 @@
 
 #: Whether thumbnails should be stored in high resolution (used by :doc:`easy-thumbnails:index`)
 THUMBNAIL_HIGH_RESOLUTION = True
+
+##############
+# Pagination #
+##############
+
+#: Number of entries displayed per pagination chunk
+#: see :attr:`django-pagination:django.core.paginator.Paginator`
+
+PER_PAGE = 25
```

Comment:
Oh, just one thought: Maybe 25 is a bit much? Since one table row has a height of ~50px, 25 rows result in a total height of 1250px. If we assume that most people have a full HD resolution with a screen height of 1080px, they cannot use the list without scrolling. I think 16-18 entries would maybe fit better?

Reply:
Yes, sounds good. I was also thinking about customization options, but it is always good to start with a default value that suits at least.:+1: